Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/SpherePointerVisual.cs

```diff
@@ -82,6 +77,11 @@ private void CheckAsset(object asset, string fieldname)
 
         public void Update()
         {
+            if (!Application.isPlaying)
```

Comment:
Can update be called when the app isn't playing?

Reply:
When a class uses the ExecuteAlways attribute Update can get called.

Reply:
This class doesn't appear to have that attribute though

Reply:
My mistake, thanks.